jupyterlab-dataops/Dockerfile

```diff
@@ -1,101 +1,34 @@
-#     unzip SimbaJDBCDriverforGoogleBigQuery42_1.2.16.1020.zip -d /tmp/cloudbeaver/server/drivers/bigquery/src/main/resources && \
-#     rm SimbaJDBCDriverforGoogleBigQuery42_1.2.16.1020.zip ; fi
-# ADD resources/cloudbeaver/plugin-${PROVIDER:-default}.xm[l] /tmp/cloudbeaver/server/bundles/io.cloudbeaver.resources.drivers.base/plugin.xml
-# ADD resources/cloudbeaver/drivers/pom-${PROVIDER:-default}.xm[l] /tmp/cloudbeaver/server/drivers/pom.xml
-# ADD resources/cloudbeaver/drivers/${PROVIDER}/pom.xm[l] /tmp/cloudbeaver/server/drivers/bigquery/pom.xml
-
-# Build
-# RUN cd /tmp/cloudbeaver/deploy && ./build.sh "" ""
-
-RUN mkdir -p $HOME && chown -R $USER_NAME $HOME && \
-    mkdir -p /opt/tools/{bin,logos} && \
-    chmod -R 755 /opt/tools/ && \
-    mkdir -p /mnt/data && chown -R $NB_USER /mnt/data
-
-# Install Python and Jupyter extensions
-RUN pip install -r $HOME/requirements.txt
-RUN if [[ -z "$PROVIDER" ]] ; then echo "Argument PROVIDER not provided" ; else pip install -r $HOME/requirements-${PROVIDER}.txt ; fi
-RUN jupyter labextension install @jupyterlab/server-proxy jupyterlab-topbar-extension
-
-# Install code server
-RUN cd /opt/ && wget https://github.com/cdr/code-server/releases/download/v4.3.0/code-server-4.3.0-linux-amd64.tar.gz && \
-    tar zxvf code-server-4.3.0-linux-amd64.tar.gz
-
-# # Copy Cloudbeaver
-# RUN mv /tmp/cloudbeaver/deploy/cloudbeaver /opt/tools/cloudbeaver
-# RUN mkdir -p /opt/tools/cloudbeaver/workspace && chmod -R 755 /opt/tools && chmod a+rwX -R /opt/tools/cloudbeaver/workspace
-
-USER $NB_USER
-RUN /opt/code-server-*-linux-amd64/bin/code-server \
-    --extensions-dir $HOME/code-server/extensions \
-    --user-data-dir $HOME/code-server/data \
-    --config $HOME/code-server/config.yaml \
-    --install-extension ms-python.python \
-    /opt/code-server-*-linux-amd64/bin/code-server \
-    --install-extension /tmp/$NB_USER/vsix/innoverio.vscode-dbt-power-user-0.5.16.vsix \
-    /opt/code-server-*-linux-amd64/bin/code-server \
-    --install-extension /tmp/$NB_USER/vsix/bastienboutonnet.vscode-dbt-0.5.1.vsix
-
-ADD resources/vsc/settings.json $HOME/code-server/data/User/settings.json
-
-# COPY resources/cloudbeaver/conf /opt/tools/cloudbeaver/conf
-# ADD resources/cloudbeaver/conf/.cloudbeaver.auto.conf /opt/tools/cloudbeaver/conf/.cloudbeaver.auto.conf
-# ADD resources/cloudbeaver/conf/cloudbeaver.conf /opt/tools/cloudbeaver/conf/cloudbeaver.conf
-# ADD resources/cloudbeaver/conf/initial-data-sources-${PROVIDER}.con[f] /opt/tools/cloudbeaver/conf/initial-data-sources.conf
-# ADD resources/cloudbeaver/conf/product.conf /opt/tools/cloudbeaver/conf/product.conf
-
-ENV HOME=/home/$USER_NAME
-ENV CODE_WORKINGDIR="${HOME}/work"
-ENV PATH="${PATH}:${CONDA_DIR}/envs/python38/bin"
-ENV CONDA_DEFAULT_ENV=python38
-ENV JUPYTER_ENABLE_LAB=yes
+FROM dbeaver/cloudbeaver:22.0.5 AS builder-dbeaver
+FROM gcr.io/deeplearning-platform-release/base-cpu:m88 as base
```

Comment:
How big this image is? I'm a little bit worried that it may increase the size a few times.

Reply:
m88 `1.1 GB`
jupyter/base-notebook:python-3.8.8 `201.75 MB`

But our build image now weight `1.8 GB` rather than `2.4 GB` where it was in `bigquery-1.0.5`